source/wp-content/themes/wporg-showcase-2022/src/site-meta-list/index.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * Block Name: Site Meta List
+ * Description: Display a site meta data as a list.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\Theme\Showcase_2022\Site_Meta_List;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Render the block content.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the block markup.
+ */
+function render( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
+		return '';
+	}
+
+	$list_items = array();
+
+	$meta_fields = array(
+		array(
+			'label' => __( 'Author', 'wporg' ),
+			'key' => 'author',
+		),
```

Comment:
It would be nice if this, and all the other code was using consistent indentation of array items. Same for variable alignments.
ie.
```suggestion
		array(
			'label' => __( 'Author', 'wporg' ),
			'key'   => 'author',
		),
```

Reply:
Right. I just let our [wporg-repo-tools](https://github.com/WordPress/wporg-repo-tools/blob/trunk/configs/phpcs.xml.dist) configuration control that. 

Running `composer run format [...]` didn't care about indentation.